lib/improver/wind_downscaling.py

```diff
@@ -506,19 +518,19 @@ def _calc_height_corr(self, u_a, heightg, mask, onemfrac):
         """Function to calculate the additive height correction.
 
         Args:
-            u_a: 2D array (float)
-                outer velocity, e.g. velocity at h_ref_orig
-            heightg: 1D or 3D array
-                heights above orography
-            mask: 3D array(logical)
-                Masks the hc_add result
-            onemfrac: currently, scalar = 1.
-                In principle, it is a function of position and height, e.g.
-                a 3D array (float)
+            u_a (np.ndarray):
+                2D array float32 - outer velocity, e.g. velocity at h_ref_orig
+            heightg (np.ndarray):
+                1D or 3D array float32 - heights above orography
+            mask (bool):
```

Comment:
Could this be changed to be:
```python
    mask (np.ndarray):
```
You could mention that the contents of the array will be boolean in the docstring.